bindings/python/test.py

```diff
@@ -73,6 +73,15 @@ def test_sha1hash(self):
 		self.assertEqual(h, str(s))
 
 
+class test_session(unittest.TestCase):
+	def test_post_session_stats(self):
+		s = lt.session({'alert_mask': lt.alert.category_t.stats_notification})
+		s.post_session_stats()
+		a = s.wait_for_alert(1000)
+		self.assertTrue(isinstance(a, lt.session_stats_alert))
```

Comment:
I guess a will be None, in case there is no alert... I wouldn't bet my life on it though :p